js/buttons.js

```diff
@@ -153,14 +153,15 @@ function updateButtonWidths(category) {
 }
 
 function updateButtons(category) {
+  
   waterUseViz.elements.buttonBox
-    .selectAll('.button rect')
+    .selectAll('.button .filled-button')
     .style('fill', function(d) {
       var col = categoryToColor(d,0);
       if(d === category) {
         col = categoryToColor(d,1);
       }
-      return (col);
+      return(col);
```

Comment:
does this work? I thought javascript was `return value;` instead of R's `return(value)`

Reply:
I just double checked and it seems to work both ways just fine. I'll take out the parenthesis for style consistencies...but yeah, it works.